src/Avalonia.Controls/Primitives/Thumb.cs

```diff
@@ -56,6 +56,26 @@ protected virtual void OnDragCompleted(VectorEventArgs e)
         {
         }
 
+        protected override void OnPointerCaptureLost(PointerCaptureLostEventArgs e)
+        {
+            if (_lastPoint.HasValue)
+            {
```

Comment:
We probably don't want to set e.Handled on CaptureLost event. As it wasn't set here.